app/models/investigation.rb

```diff
@@ -53,4 +53,22 @@ def positioned_studies
   def self.user_creatable?
     Seek::Config.investigations_enabled
   end
+
+  # utility method for cleaning up durng testing, not to be used in production code
+  def deep_destroy!
```

Comment:
I've tried using `deep_destroy!` during the review, but I noticed that the data files and samples are not being destroyed after running this method.

Reply:
Updated and fixed. Also extended to delete samples and data files associated with observation units, which lead to an issue deleting `ObservationUnitAsset` which has also been fixed